src/components/Scale.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react'
```

Comment:
This control is overlaying the existing zoom control:

![image](https://user-images.githubusercontent.com/695720/74146871-5a7cdf00-4c02-11ea-894b-71eae20d8092.png)

Reply:
Ok, sounds good.